### ISSUE TITLE
Accessibility concerns

### ISSUE THREAD

**Issue by sbhat98** [HackGT/hackgt7-website#26]
It's not possible to access all of the information on this site through keyboard navigation. For the FAQs, there's no way to focus on the elements and open them up. This is a problem for users who cannot physically use a mouse easily, or for those whose website scrolling experience generally doesn't involve a mouse.

Also, this doesn't effect me, but all of the images on the site have the same copied alt text. For screen reader users, this can be confusing.

**Comment by 1ntEgr8** [comment]
yea hahaha, i saw that too; was about to push a fix
